clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/agent/KubernetesInstanceCachingAgent.groovy

```diff
@@ -82,8 +82,8 @@ class KubernetesInstanceCachingAgent implements  CachingAgent, AccountAware {
 
     Map<String, MutableCacheData> cachedInstances = MutableCacheData.mutableCacheMap()
 
-    pods.each { pod ->
-      def rc = pod.metadata.labels[KubernetesUtil.REPLICATION_CONTROLLER_LABEL] ?: Keys.DETACHED_POD
+    for (Pod pod : pods) {
+      def rc = pod.metadata.labels[KubernetesUtil.REPLICATION_CONTROLLER_LABEL] ?: ''
```

Comment:
Whoops, it's not, I'll delete it.

Groovy bug; we've seen this before where a closure sometimes can't access certain class attributes. In this case, `accountName` was null inside the loop, but correct outside it.